test/partitioned_methods_tests.jl

```diff
@@ -289,9 +318,18 @@ sim = test_convergence(dts,prob,SofSpa10(),dense_errors=true)
 dts = big"1.0"./big"2.0".^(5:-1:1)
 prob_big = DynamicalODEProblem(ff_harmonic_nip,big"1.0",big"0.0",
                                  (big"0.",big"70."))
+sim = test_convergence(dts,prob_big,DPRKN43T(),dense_errors=true) ### TODO
+@test sim.𝒪est[:l2] ≈ 4 rtol = 1e-1
+@test sim.𝒪est[:L2] ≈ 4 rtol = 1e-1
+sim = test_convergence(dts,prob_big,DPRKN54T(),dense_errors=true) ### TODO
+@test sim.𝒪est[:l2] ≈ 4 rtol = 1e-1
```

Comment:
?

Reply:
Just an easier way to find the broken tests in Juno.
Will delete as they get fixed.